include/mbedtls/x509_crt.h

```diff
@@ -348,6 +459,30 @@ int mbedtls_x509_crt_parse_file( mbedtls_x509_crt *chain, const char *path );
 int mbedtls_x509_crt_parse_path( mbedtls_x509_crt *chain, const char *path );
 #endif /* MBEDTLS_FS_IO */
 
+/**
+ * \brief          Parses a subject alternative name raw item
+ *                 to an identified structure;
+ *
+ * \param san_raw  The buffer hlding the raw data of the subject alternative name.
+ *
+ * \param san      The parsed subject alternative name.
```

Comment:
Minor: `The target structure to populate with the parsed presentation of the subject alternative name encoded in \p san_raw`.